### ISSUE TITLE
How to postpone signal A until signal B completes?

### ISSUE THREAD

**Issue by onmyway133** [ReactiveCocoa/ReactiveCocoa#1936]
I want to receive events from signal A, but for some reason I must wait until signal B completes (some work must be done before)

I see that `- (RACSignal *)sample:(RACSignal *)sampler` achieves similar to what I want, but not. 

> Sends the latest value from the receiver only when `sampler` sends a value

It only sends value whenever `sampler` sends a value. In my case, I want it to send value after `sampler` finishes

I find that `+ (RACSignal *)combineLatest:(id<NSFastEnumeration>)signals reduce:(id (^)())reduceBlock` solves my problem. Not sure if there are better ways

**Comment by Tiger6688** [comment]
how about:
[RACSignal concat:@[A,B]]

OR
[A then B]

**Comment by neilpa** [comment]
Closing, but please re-open if that doesn't answer your question.
